Orange/widgets/tests/base.py

```diff
@@ -300,24 +411,25 @@ def test_parameters_default(self):
         self.widget.apply_button.button.click()
         if hasattr(self.widget.learner, "params"):
             learner_params = self.widget.learner.params
-            for element in self.gui_to_params:
+            for element in self.parameters:
```

Comment:
I would rename this to

    for parameter in self.parameters:

to match similar for in test_parameters method